TOMP-API.yaml

```diff
@@ -1015,13 +1045,15 @@ paths:
           description: start of the selection
           required: true
           schema:
-            $ref: "#/components/schemas/timestamp"
+            type: string
+            format: date-time
         - name: to
```

Comment:
why not place it centrally? Now you provide in future versions that on different places different notations can be used.